clang/test/SemaHLSL/BuiltIns/normalize-errors.hlsl

```diff
@@ -0,0 +1,31 @@
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm -disable-llvm-passes -verify -verify-ignore-unexpected

```

Comment:
Hi @bob80905 
I think this "-emit-llvm -disable-llvm-passes" will make the test write to the local filesystem?
Can we try to avoid that, e.g. by just removing "-emit-llvm"?
See e.g. b7730a23efb2 which fixed a similar case recently.